ledger/participant-integration-api/src/main/scala/platform/apiserver/execution/StoreBackedCommandExecutor.scala

```diff
@@ -77,7 +77,7 @@ private[apiserver] final class StoreBackedCommandExecutor(
               commands.actAs.toList,
               commands.applicationId.unwrap,
               commands.commandId.unwrap,
-              commands.deduplicateUntil,
+              commands.submittedAt.plus(commands.deduplicationDuration),
```

Comment:
Worth a small util?

Reply:
Yeah, why not?

Reply:
Done. I called it `deduplicateUntil` to stay aligned with its usages, though I don't particularly like the name. I may change it globally in a future changeset.